sdv/sequential/par.py

```diff
@@ -540,9 +545,15 @@ def sample_sequential_columns(self, context_columns, sequence_length=None):
                 set(context_columns.columns), set(self._context_synthesizer._model.columns)
             )
         )
+
+        datetime_columns = self._get_context_datetime_columns()
+        if datetime_columns:
+            context_columns[datetime_columns] = self._data_processor.transform(
+                context_columns[datetime_columns]
+            )
```

Comment:
Can we pull this into a function to transform context columns that are transformed by the PAR dataprocessor?